### PR TITLE
UIP-2806: Release over_react 1.19.0

### PR DESCRIPTION
**Pull request by jacehensley-wf**
Pulls Included in Release:
* [Add BuiltReduxUiComponent](https://github.com/Workiva/over_react/pull/118)
* [UIP-2889 Use dependency_validator](https://github.com/Workiva/over_react/pull/126)

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/over_react/compare/1.18.1...jacehensley-wf:release_1_19_0
Diff Between Last Tag and New Tag: https://github.com/Workiva/over_react/compare/1.18.1...1.19.0

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+## 1.19.0
+
+> [Complete `1.19.0` Changeset](https://github.com/Workiva/over_react/compare/1.18.1...1.19.0)
+
+__New Features__
+* [#118] Add `BuiltReduxUiComponent` and `BuiltReduxUiProps`.
+
+__Tech Debt__
+* [#126]: Use dependency_validator.
+
```
Comment (greglittlefield-wf) [inline CHANGELOG.md]: Can we also add a note about the updated SDK version constraint?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # OverReact Changelog
 
+## 1.19.0
+
+> [Complete `1.19.0` Changeset](https://github.com/Workiva/over_react/compare/1.18.1...1.19.0)
+
+__New Features__
+* [#118] Add `BuiltReduxUiComponent` and `BuiltReduxUiProps`.
```
Comment (greglittlefield-wf) [inline CHANGELOG.md]: We should probably mention here that these are considered unstable and found in `package:over_react/experimental.dart`

**Comment by greglittlefield-wf** [comment]
+1

**Comment by garywhitlock-wf** [comment]
@jacehensley-wf Can you let @kaitlynchilders-wk know when this is merged?  We will be bumping the version of over_react in DT tomorrow.

**Comment by jayudey-wf** [comment]
QA Resource Approval: +1

- [x] Dev +1's only needed for Release ticket

Ready for merge and tag.

@Workiva/release-management-pp

**Pull request merged**